README.md

```diff
@@ -16,22 +16,24 @@ The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the
 * Update the version of `{replace_version page.url 'X.0.0.X' }` in [src/main/content/antora_ui/src/partials/head-info.hbs](https://github.com/OpenLiberty/openliberty.io/blob/staging/src/main/content/antora_ui/src/partials/head-info.hbs) so that the canonical meta tag is set to the latest version of the documentation for all versions of the docs. 
 * Update the version of the url `<loc>https://openliberty.io/docs/X.0.0.X/overview.html</loc>` in the [src/main/content/sitemap.xml](https://github.com/OpenLiberty/openliberty.io/blob/staging/src/main/content/sitemap.xml).
 
-7. Before proceeding, make sure all of the pull requests and changes from previous steps have been merged into their respective repos. Make a pull request from your branch into the `staging` branch. Once the pull request has been reviewed merge it in. The [staging site](http://dev-openlibertyio.mybluemix.net/) should automatically rebuild once the pull request has merged in. It will take ~40 minutes for the changes to show up on the staging site. You can track the progress using the [IBM Cloud staging pipeline](https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350?env_id=ibm:yp:us-south). Test that the docs link works and redirects to the newest doc version once the site is done building.
+1. Before proceeding, make sure all of the pull requests and changes from previous steps have been merged into their respective repos. Make a pull request from your branch into both the `draft` and`staging` branches. Once the pull requests have been reviewed merge them in. The [staging site](https://staging-openlibertyio.mybluemix.net/) should automatically rebuild once the pull request has merged in. It will take ~40 minutes for the changes to show up on the staging site. You can track the progress using [Travis](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches). Test that the docs link works and redirects to the newest doc version once the site is done building.
 
-8. Make a pull request from the [UI staging branch](https://github.com/OpenLiberty/openliberty.io/tree/staging) into the [UI prod branch](https://github.com/OpenLiberty/openliberty.io/tree/prod). The [openliberty.io site](https://openliberty.io/) will automatically rebuild when the pull request is merged in. You can track the progress using the IBM Cloud pipeline(https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south).
+2. Make a pull request from the [UI staging branch](https://github.com/OpenLiberty/openliberty.io/tree/staging) into the [UI prod branch](https://github.com/OpenLiberty/openliberty.io/tree/prod). The [openliberty.io site](https://openliberty.io/) will automatically rebuild when the pull request is merged in. You can track the progress using Travis(https://travis-ci.com/github/OpenLiberty/openliberty.io/branches).
```

Comment:
This doesn't seem right.... Travis shouldn't know anything about the process of building the `prod` site?
The previous version that pointed to IBM Cloud looks correct